src/styles/blocks/cards.scss

```
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $white-color;
+
+  &__screen {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    width: 166px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 36px 0 0;
+    text-align: left;
+  }
+
+  &__code {
+    font-size: 10px;
+    text-align: left;
+    color: $gray-color;
+    margin-top: 0;
+    margin-bottom: 16px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    color: $gray-color;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-price {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__btn {
+    width: 166px;
```

```suggestion
    width: 100%;
```
Avoid hardcoding width of the button